pkg/controller/ground/bootstrap/dns/manifests.go

```diff
@@ -143,8 +146,8 @@ spec:
         args:
         - --v=2
         - --logtostderr
-        - --probe=kubedns,127.0.0.1:10053,kubernetes.default.svc.{{ .Domain }},5,A
-        - --probe=dnsmasq,127.0.0.1:53,kubernetes.default.svc.{{ .Domain }},5,A
+        - --probe=kubedns,127.0.0.1:10053,kubernetes.default.svc.{{ .Domain }},5,SRV
```

Comment:
Is this backward compatible and won't break cluster that are created with a version <1.12

Reply:
That change is in since 2years/1.9.0. It's wierd that it works with the `A` record. According to the docs kube-dns creates a `SRV` entry for headless services, which the `kubernetes.default` should be. https://github.com/kubernetes/kubernetes/commit/3e4b9fad6a4babf0311ffca6aa438d04464bb71c